source/includes/fact-timezone-description.rst

```diff
@@ -0,0 +1,30 @@
+``Optional.`` The timezone of the operation result.
+``<tzExpression>`` must be a valid :ref:`expression
+<aggregation-expressions>` that resolves to a string formatted as either
+an `Olson Timezone Identifier
+<https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>`_ or a
+`UTC Offset <https://en.wikipedia.org/wiki/List_of_UTC_time_offsets>`_.
+If no ``timezone`` is provided, the result is displayed in ``UTC/GMT``.
```

Comment:
Note: UTC is a _time standard_ and GMT is a _UK time zone_. When daylight savings time is in effect in the UK, they go to _BST (British Standard Time)_. You should refer to this as UTC only. (https://www.timeanddate.com/time/gmt-utc-time.html)